IBM/carbon-addons-iot-react
# [SideNav] 1082 sidenav child items fix

Closes #1082 

**Summary**

- Styling changes to fix second tier children from getting cut off

**Change List (commits, features, bugs, etc)**

- Added styles to src/components/SideNav/_side-nav.scss
- Added to story to show more second tier children
- updates snapshots 

**Acceptance Test (how to verify the PR)**

- go to SideNav story 
- Click on dashboards
- see multiple second tier children
